corpus/config/models.py

```diff
@@ -3,6 +3,24 @@
 DATETIME_FORMAT = "%d-%m-%Y %H:%M:%S"
 
 
+class SIG(models.Model):
```

Comment:
Let's do one thing - lets add a [`SlugField`](https://docs.djangoproject.com/en/5.0/ref/models/fields/#slugfield) to the model, and let that be like the name of the sig, but in lowercase.

Then, your url will look something like `/sig/<str:sig_slug>`, where the slug can be `compsoc`, `diode` and `piston` (all lowercase, so that it looks better and conforms to most web standards).

Then in the view, you can query by slug and get the appropriate model object. Make sure that you add `unique=True` to the `SlugField` as well.

Reply:
but one concern, when I fetch say {{sig.name}}, then I'll get `compsoc` instead of `CompSoc` on my sig.html page right?
So would that be alright?

Reply:
done